cmd/vct/startcmd/start.go

```diff
@@ -130,6 +136,16 @@ const (
 		" Alternatively, this can be set with the following environment variable: " + tlsServeKeyPathFlagEnvKey
 	tlsServeKeyPathFlagEnvKey = envPrefix + "TLS_SERVE_KEY"
 
+	tlsMetricsServeCertPathFlagName  = "tls-metrics-serve-cert"
+	tlsMetricsServeCertPathFlagUsage = "Path to the server certificate to use when serving metrics HTTPS." +
+		" Alternatively, this can be set with the following environment variable: " + tlsMetricsServeCertPathEnvKey
+	tlsMetricsServeCertPathEnvKey = envPrefix + "TLS_METRICS_SERVE_CERT"
+
+	tlsMetricsServeKeyPathFlagName  = "tls-metrics-serve-key"
```

Comment:
We don't need TLS for metrics since this port isn't going to be exposed publicly.

Reply:
ok, will remove